packages/client/wallets/smart-wallet/src/utils/constants.ts

```diff
@@ -4,28 +4,8 @@ export const DATADOG_CLIENT_TOKEN = "pub035be8a594b35be1887b6ba76c4029ca";
 export const CROSSMINT_DEV_URL = "http://localhost:3000/api";
 export const CROSSMINT_STG_URL = "https://staging.crossmint.com/api";
 export const CROSSMINT_PROD_URL = "https://www.crossmint.com/api";
-export const ZD_ETHEREUM_PROJECT_ID = "9ee29857-8077-404b-9a9a-31eeea996a4a";
-export const ZD_POLYGON_PROJECT_ID = "023d4a21-d801-4450-b629-24439ab1369d";
-export const ZD_BSC_PROJECT_ID = "3d166617-da86-494b-9348-e8a13343bc04";
-export const ZD_OPTIMISM_PROJECT_ID = "e9314f9e-a13d-414f-b965-c591a0248243";
-export const ZD_ARBITRUM_PROJECT_ID = "1641cd99-c1ef-404a-9d26-a9dc67b1ba51";
-export const ZD_GOERLI_PROJECT_ID = "3cfecfb6-9d7d-4ef6-acaa-ac8f79f6cd5a";
-export const ZD_SEPOLIA_PROJECT_ID = "7ff22858-06f0-4f3a-8b46-5b41d8c75d0e";
-export const ZD_AMOY_PROJECT_ID = "3b24773b-d91e-4c01-8ce5-04807463bbca";
-export const ZD_ZKATANA_PROJECT_ID = "d54706a0-304b-419e-8a33-03c26ba3f0e9";
-export const ZD_ZKYOTO_PROJECT_ID = "ce986d52-4f27-4a2b-b429-eb2322f15f32";
-export const ZD_ASTAR_PROJECT_ID = "6204f336-643f-41af-94e1-f8d146c91675";
-export const ZD_BASE_SEPOLIA_PROJECT_ID = "3eb830c5-f91b-48e0-bb7d-dc30103a60b2";
-export const ZD_OPTIMISM_SEPOLIA_PROJECT_ID = "6188b92b-993e-4f39-be22-56e4806416a8";
-export const ZD_ARBITRUM_SEPOLIA_PROJECT_ID = "8b8b6d1a-184c-4198-8f29-c07e63aad595";
-export const ZD_BASE_PROJECT_ID = "5535aa3b-4f9c-45af-9c38-0072369564a3";
-export const ZD_ARBITRUM_NOVA_PROJECT_ID = "b6a6db7c-65de-4f74-9d8f-f70d9c083531";
-export const RELAY_API_KEY = "c8999b7e-8bb7-416d-a854-dac8632f9ee6_crossmint";
 export const SCW_SERVICE = "SCW_SDK";
 export const SDK_VERSION = "0.1.0";
 export const API_VERSION = "2024-06-09";
 export const BUNDLER_RPC = "https://rpc.zerodev.app/api/v2/bundler/";
 export const PAYMASTER_RPC = "https://rpc.zerodev.app/api/v2/paymaster/";
-export const PM_BASE_SEPOLIA_RPC =
-    "https://api.developer.coinbase.com/rpc/v1/base-sepolia/6BTAmOQZ0x1YWtI24hIKKqdHWmVP1UXf";
-export const PM_BASE_RPC = "https://api.developer.coinbase.com/rpc/v1/base/6BTAmOQZ0x1YWtI24hIKKqdHWmVP1UXf";
```

Comment:
These didn't work!

The free gas is nice, but falling back to ZD for the EAP release.